lib/eaco/rake/default_task.rb

```diff
@@ -40,7 +40,6 @@ def initialize
             run_specs
             run_cucumber
             report_coverage
-            generate_documentation
```

Comment:
OK, I have added #6 asking for a revert, with some details that'll aid investigation.